torchbenchmark/util/env_check.py

```diff
@@ -172,9 +172,12 @@ def deterministic_torch_manual_seed(*args, **kwargs):
 
         seed = MAIN_RANDOM_SEED
         import torch.cuda
-
         if not torch.cuda._is_in_bad_fork():
             torch.cuda.manual_seed_all(seed)
+
+        import torch.xpu
```

Comment:
I have one question, sometimes `torch.xpu` is not available, do we need to wrap it in a try block? cc @retonym

Reply:
Hi, I tried the code on a non-xpu device, and it works.
The code here is same as the pytorch repo: https://github.com/pytorch/pytorch/blob/main/torch/random.py#L51-L53